wdl/ExpansionHunter.wdl

```diff
@@ -20,6 +20,8 @@ workflow ExpansionHunter {
         File reference_fasta
         File? reference_fasta_index
         File variant_catalog
+        String? individual_id
```

Comment:
```suggestion
        String? sample_id
```
Just so it's crystal clear to the user, since `individual` could be construed as a slightly different entity under a legacy Terra data model.

Also, can you add a comment that the sample id and ped file must be provided together?